models/public/midasnet/model.yml

```
@@ -36,9 +36,9 @@ files:
     sha256: ba87812d0b5b247966471511d2d1a47fd6f492afb372bc49af6de3a46f4cf6c7
     source: https://raw.githubusercontent.com/intel-isl/MiDaS/ffb70fd13361434114383ce7eb898c2f5bec6176/models/blocks.py
   - name: model.pt
-    size: 422397766
-    sha256: 5237e007f53f05d0a66f40590abfead08dbb662629089e3462daf741ac9932d6
-    source: https://download.01.org/opencv/public_models/032020/midasnet/model-001.pt
+    size: 422509849
+    sha256: 933c46ec14e4041122852d8d0acf822ff9f4b94fd07f0ff42fa593f135d57cab
+    source: TODO PLEASE UPDATE https://download.01.org/opencv/public_models/032020/midasnet/model-001.pt WITH NEW VERSION OF https://drive.google.com/file/d/1Jf7qRG9N8IW8CaZ7gPisO5RtlLl63mNA
```

You can use Google drive as file location. You just need replace this line with
```suggestion
    source:
      $type: google_drive
      id: 1Jf7qRG9N8IW8CaZ7gPisO5RtlLl63mNA
```

---

We had this previously and this created problems; see the note in [midasnet.md](https://github.com/openvinotoolkit/open_model_zoo/blob/617f34c7f05026a4523e1e10d519fb96cddb806c/models/public/midasnet/midasnet.md) as well as [this commit](https://github.com/openvinotoolkit/open_model_zoo/commit/1bc82be124dcd5e24822620e95b63a93b0ee5a81#diff-a331ee1f9b92e94f27c206566b4c8dd59f7ef238a3a7bb09425a5d8d6a804ae7).

The file size is quite big (> 400 MB). This causes the gdrive to (sometimes) ask for "Do you really want to download" or sth. like this, and this did not work with the download script for some users (including me).

---

We use the link to github now.